spec/models/comfy/cms/page_spec.rb

```diff
@@ -234,6 +234,16 @@
       expect(results.map(&:page_views)).to eq([200, 152, 100])
     end
 
+    it 'should not return articles with zero page_views' do
+      create :page, page_views: 1, position: 1
+      create :page, page_views: 0, position: 2
+      create :page, page_views: 0, position: 3
+
+      results = Comfy::Cms::Page.most_popular(3)
+
+      expect(results.map(&:page_views)).to eq([1])
+    end
+
```

Comment:
I think contexts are a way to keep contexts together :), and this was in the contexts of the #most_popular scope. Having a context for each 'it' block seems excessive. But saying that I like the way it came out so :+1: